packages/core/src/services/documentLogs/computeDocumentLogs.ts

```diff
@@ -11,41 +15,33 @@ import {
   runErrors,
   workspaces,
 } from '../../schema'
-import { getCommonQueryConditions } from './computeDocumentLogsWithMetadata'
+import { getDocumentLogFilters } from './computeDocumentLogsWithMetadata'
 
 export function computeDocumentLogsQuery(
   {
     workspaceId,
     documentUuid,
-    draft,
-    allowAnyDraft,
     page = '1',
     pageSize = String(DEFAULT_PAGINATION_SIZE),
+    filterOptions,
   }: {
     workspaceId: number
     documentUuid?: string
-    draft?: Commit
-    allowAnyDraft?: boolean
     page?: string
     pageSize?: string
+    filterOptions?: DocumentLogFilterOptions
   },
   db = database,
 ) {
   const repo = new DocumentLogsRepository(workspaceId, db)
   const offset = calculateOffset(page, pageSize)
+  const conditions = [
+    isNull(runErrors.id),
```

Comment:
I think `eq(workspaces.id, workspaceId),` is missing?

Reply:
Is in repo no?

Reply:
![image](https://github.com/user-attachments/assets/69a1bcb4-487d-418f-b06e-0ba970ddc561)